### ISSUE TITLE
`read` should allow for files without `.json` extensions

### ISSUE THREAD

**Issue by rconstanzo** [flucoma/flucoma-max#270]
Since all the `fluid.dataset~` objects write files without extensions (by default), the corresponding `read` messages should let you load any file (sans extension).

and/or

The `write` command should bring up the file menu that allows you to select the filetype to be written (ala `buffer~`).

![Screenshot 2022-06-14 at 9 04 51 pm](https://user-images.githubusercontent.com/2747015/173678851-96b7fe61-9096-47f6-82cb-0c950c3616a8.png)

![Screenshot 2022-06-14 at 9 06 12 pm](https://user-images.githubusercontent.com/2747015/173679059-77b9c1ef-3e9e-4b02-8a34-9583d777c12c.png)

![Screenshot 2022-06-14 at 9 05 05 pm](https://user-images.githubusercontent.com/2747015/173678883-9ab86103-8cb8-42b5-9111-764e8a647cbb.png)

Basically working on a set of tools for beginner-ish use and it's an odd point of friction where the default files you can `write` are not the files you can then `read`.

**Comment by jamesb93** [comment]
> Since all the `fluid.dataset~` objects write files without extensions (by default), the corresponding `read` messages should let you load any file (sans extension).

I don't think this is the case.

On macOS you should see this:

![image](https://user-images.githubusercontent.com/13281862/174633874-84b63b42-5b9c-4bcd-a306-060476f279fa.png)

**Comment by jamesb93** [comment]
> > > Since all the `fluid.dataset~` objects write files without extensions (by default), the corresponding `read` messages should let you load any file (sans extension).
> > 
> > 
> > I don't think this is the case.
> > On macOS you should see this:
> > ![image](https://user-images.githubusercontent.com/13281862/174633874-84b63b42-5b9c-4bcd-a306-060476f279fa.png)
> 
> I definitely don't see that. I get what you see in the first screenshot above.

What version of macOS?

**Comment by rconstanzo** [comment]
Mojave.

Just checked on my laptop (Monterey) and it gives me the `.json` prompt.

**Comment by weefuzzy** [comment]
If the same code from us (which goes through Max API calls) is resulting in different OS prompts, I don't hold out much hope for us being able to sway what it does
